src/plugins/charts/BarGraphCompositionPolicy.js

```diff
@@ -23,10 +23,10 @@
 import { BAR_GRAPH_KEY } from './BarGraphConstants';
 
 export default function BarGraphCompositionPolicy(openmct) {
-    function hasRange(metadata) {
+    function hasNonEnumRange(metadata) {
         const rangeValues = metadata.valuesForHints(['range']);
 
-        return rangeValues.length > 0;
+        return rangeValues && rangeValues.length > 0 && rangeValues[0].format !== 'enum';
```

Comment:
Perhaps what we should do is check that there is at least 1 range metadata that is Not and enum.
And then in the Vue file, we should filter out any range metadata that is an enum.

Reply:
If a domain object's range is just:
```
[
    {
        "key": "state",
        "source": "output",
        "name": "State",
        "format": "enum",
        "enumerations": [
            {
                "string": "Foo",
                "value": 0
            },
            {
                "string": "Default",
                "value": 1
            }
        ],
        "hints": {
            "range": 1,
            "priority": 1
        },
        "values": [
            0,
            1
        ],
        "max": null,
        "min": null
    },
    {
        "key": "output",
        "name": "Value",
        "format": "string",
        "hints": {
            "range": 2,
            "priority": 2
        },
        "source": "output"
    }
]
```
Ce should be excluding the first enum, but only keeping the string? Or should we be excluding both?